src/timestamp.rs

```diff
@@ -48,11 +61,56 @@ impl Timestamp {
     pub fn get_diff_duration(&self, other: &Timestamp) -> Duration {
         (self.time - other.time).to_duration()
     }
+
+    /// Convert to a RFC3339 time representation with nanoseconds precision.
+    /// e.g.: `"2024-07-01T13:51:12.129693000Z/33"``
+    #[cfg(feature = "std")]
+    pub fn to_string_rfc3339(&self) -> String {
```

Comment:
Should the function be called `to_string_rfc3339_lossy` To reflect the fact the conversion is lossy?